teuthology/task/buildpackages/common.sh

```diff
@@ -41,9 +41,6 @@ function get_ceph() {
 
     test -d ceph || git clone ${git_ceph_url} ceph
     cd ceph
-    if test -d src ; then # so we don't try to fetch when using a fixture
-       git fetch --tags http://github.com/ceph/ceph
-    fi
```

Comment:
keeping it by default with an environment variable to disable it would be a good way to control it in the rare case where it needs to be changed. Fixing the problem would mean changing how github & git clone and push by default: it's not going to happen any time soon ;-)